GDJS/Runtime/pixi-renderers/pixi.ts

```diff
@@ -0,0 +1,118 @@
+// The custom PIXI.js bundle for GDevelop
+// Unused functionality like the interactions plugins is removed for performance reasons.
+// Add an export for any new additional PIXI feature you might need, and regenerate the types.
+
+// Core modules
+export * from '@pixi/core';
+export * from '@pixi/constants';
+export * from '@pixi/math';
+export * from '@pixi/settings';
+import * as utils from '@pixi/utils';
+export { utils };
+
+// Core plugins
+export * from '@pixi/canvas-renderer';
+
+export * from '@pixi/display';
+import '@pixi/canvas-display';
+
+export * from '@pixi/prepare';
+export * from '@pixi/canvas-prepare';
+
+export * from '@pixi/extract';
+export * from '@pixi/canvas-extract';
+
+// Base objects
+export * from '@pixi/sprite';
+export * from '@pixi/canvas-sprite';
+
+export * from '@pixi/sprite-tiling';
+import '@pixi/canvas-sprite-tiling';
+
+export * from '@pixi/text';
+import '@pixi/canvas-text';
+
+export * from '@pixi/particle-container';
+import '@pixi/canvas-particle-container';
+
+export * from '@pixi/graphics';
+export * from '@pixi/canvas-graphics';
+//import '@pixi/graphics-extras';
```

Comment:
not needed?

Reply:
I checked and while it was imported by the shape painter object, the methods it adds are unused. Removing it now :+1: